versionedRouter.js

```diff
@@ -100,6 +104,37 @@ async function handleDest(ctx, version, destination) {
   ctx.set("apiVersion", API_VERSION);
 }
 
+async function routerHandleDest(ctx) {
+  const { destType, input } = ctx.request.body;
+  const routerDestHandler = getRouterDestHandler(destType);
```

Comment:
We can reuse the `getDestHandler` method with the constant value `v0` instead of the new method.

Reply:
done.